lib/image-stream/index.js

```diff
@@ -67,15 +67,15 @@ const errors = require('../shared/errors');
  * });
  */
 exports.getFromFilePath = (file) => {
-  return utils.getArchiveMimeType(file).then((type) => {
-    if (!_.has(handlers, type)) {
-      throw errors.createUserError('Invalid image', `The ${type} format is not supported`);
-    }
+  const type = utils.getArchiveMimeType(file);
```

Comment:
No need to use a Promise here anymore as `. getArchiveMimeType()` isn't async anymore